### DISCUSSION TITLE
Proposal to add documentation in another languages

### DISCUSSION THREAD

**Discussion by Daniel-Alvarenga** [zed-industries/zed#16095] [Show and tell]
The project documentation is only in English, which may make access difficult for people who do not speak the language. Since Zed is used by a global community, this may be a barrier for some contributors.

## Proposal

I suggest adding Spanish and Mandarin versions of key documents, and I'm willing to contribute to that. This change can:

- Make the project more accessible to more people.
- Increase clarity of documentation for non-English speakers.
- Attract more contributors from other parts of the world.

Pros:
- More people can participate and understand the project.
- More diverse and engaged community.

Cons:
- It requires effort to translate and keep translations up to date.

## Considerations
I am willing to help with translations and maintenance. I know this may take a little more work, but I believe the benefits to the global Zed community are worth it. I would like to hear what you think and how we can move forward with this idea.

**Comment by hwhsu1231** [comment]
An implementation of this proposal: https://github.com/zed-industries/zed/discussions/45647
